main.js

```diff
@@ -58,6 +60,18 @@ function checkLocalStorage() {
 }
 
 //form validation
+function checkDigits(event) {
+  var validCodes = [39, 37, 8, 9, 91, 13, 38, 40];
+
+  if (event.keyCode >= 48 && event.keyCode <= 57 || validCodes.includes(event.keyCode)) {
```

Comment:
For readability, I would add parentheses around the and conditional. So `(event.keyCode >= 48 && event.keyCode <= 57) || validCodes.includes(event.keyCode)`.